data/words/verbs_with_conjugaisons.json

```diff
@@ -0,0 +1 @@
,"bury","bury","bury"],"perfecto":["buried","buried","buried","buried","buried","buried"],"imperfect":["buried","buried","buried","buried","buried","buried"],"pluscuamperfecto":["buried","buried","buried","buried","buried","buried"],"futur":["bury","bury","bury","bury","bury","bury"],"Futuro anterior":["buried","buried","buried","buried","buried","buried"]},"subjuntive":{"Presente":["bury","bury","bury","bury","bury","bury"],"Perfecto":["buried","buried","buried","buried","buried","buried"]},"condicional":{"Condicional":["bury","bury","bury","bury","bury","bury"],"Condicional perfecto":["buried","buried","buried","buried","buried","buried"]},"imperative":["bury","bury","bury"]},{"verb":"call","infinitive":["call"],"participio":["called"],"gerundif":["calling"],"indicative":{"present":["call","call","calls","call","call","call"],"perfecto":["called","called","called","called","called","called"],"imperfect":["called","called","called","called","called","called"],"pluscuamperfecto":["called","called","called","called","called","called"],"futur":["call","call","call","call","call","call"],"Futuro anterior":["called","called","called","called","called","called"]},"subjuntive":{"Presente":["call","call","call","call","call","call"],"Perfecto":["called","called","called","called","called","called"]},"condicional":{"Condicional":["call","call","call","call","call","call"],"Condicional perfecto":["called","called","called","called","called","called"]},"imperative":["call","call","call"]},{"verb":"camp","infinitive":["camp"],"participio":["camped"],"gerundif":["camping"],"indicative":{"present":["camp","camp","camps","camp","camp","camp"],"perfecto":["camped","camped","camped","camped","camped","camped"],"imperfect":["camped","camped","camped","camped","camped","camped"],"pluscuamperfecto":["camped","camped","camped","camped","camped","camped"],"futur":["camp","camp","camp","camp","camp","camp"],"Futuro anterior":["camped","camped","camped","camped","camped","camped"]},"subjuntive":{"Presente":["camp","camp","camp","camp","camp","camp"],"Perfecto":["camped","camped","camped","camped","camped","camped"]},"condicional":{"Condicional":["camp","camp","camp","camp","camp","camp"],"Condicional perfecto":["camped","camped","camped","camped","camped","camped"]},"imperative":["camp","camp","camp"]},{"verb":"care","infinitive":["care"],"participio":["cared"],"gerundif":["caring"],"indicative":{"present":["care","care","cares","care","care","care"],"perfecto":["cared","cared","cared","cared","cared","cared"],"imperfect":["cared","cared","cared","cared","cared","cared"],"pluscuamperfecto":["cared","cared","cared","cared","cared","cared"],"futur":["care","care","care","care","care","care"],"Futuro anterior":["cared","cared","cared","cared","cared","cared"]},"subjuntive":{"Presente":["care","care","care","care","care","care"],"Perfecto":["cared","cared","cared","cared","cared","cared"]},"condicional":{"Condicional":["care","care","care","care","care","care"],"Condiciona
```

Comment:
Can the keys like `condicional`, `subjuntive`, `pluscuamperfecto` etc., be translated to English?

`undefineded` should be `undefined`.